src/xarray_regrid/regrid.py

```diff
@@ -86,24 +86,20 @@ def conservative(
         latitude_coord: str | None = None,
         time_dim: str | None = "time",
         skipna: bool = True,
-        nan_threshold: float = 0.0,
+        nan_threshold: float = 1.0,
```

Comment:
This was just a mistake, had it as 1.0 in all the other functions but not the top level method. This matches xesmf's default.